### PR TITLE
Add contact form for Portfolio

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
**In this PR, I did the following:**

- Added HTML layout for the contact section of the page
- Added Style to the form section and used flexbox
- Applied pair programming with partner
- Style applied according to the Figma template
- Cleared linter errors
- updated the README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -208,6 +208,19 @@ <h4 class="social-links">LET’S CONNECT</h4>
...
+  <section class="contact">
+    <div class="form-title">
+      <h1>Contact me</h1>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form class="form" action="https://formspree.io/f/meqwkove" method="post"></form>
+      <input type="text" name="first_name" maxlength="30" required placeholder="  Your Name">
+      <input type="email" name="email" id="email" required placeholder="  Your Email Address">
+      <textarea name="message" id="text" maxlength="500" required placeholder="Write your message here"></textarea>
+      <button type="submit">Get in touch</button>
```
Comment (ABDULALI3468) [inline index.html:220]: Please consider wrapping all the form elements in the form tag. Currently, the form element is not holding any other elements which are making the form submission not possible.

## Consequences:-

As your form is not submitting so none of the validations are working obviously and that's why I am not able to verify them as a Code Reviewer. So consider them also when you do the changes to your form to make all of them work.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @Jawad-Nazari , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Jawad-Nazari) [reply index.html]: Hi **@ABDULALI3468** 

Thanks for your time and for reviewing my code. I have fixed the form issue. 🙏

File: style.css [hunk 2]
```diff
@@ -399,3 +399,88 @@ ul {
...
+  width: 142px;
+  padding: 12px, 16px, 12px, 16px;
+  border-radius: 8px;
+  margin-left: 24px;
+}
+
+button:hover {
+  background-color: #7f8cff;
+  color: white;
+}
```
[the remark this answers is not in the document]
Comment (Jawad-Nazari) [reply style.css]: **Hi @ABDULALI3468 ,**

Thanks for your time and for reviewing my code. I have fixed the background issue and other styles according to Figma template. 🙏

File: index.html [hunk 3]
```diff
@@ -208,6 +208,19 @@ <h4 class="social-links">LET’S CONNECT</h4>
...
+  <section class="contact">
+    <div class="form-title">
+      <h1>Contact me</h1>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form class="form" action="https://formspree.io/f/meqwkove" method="post">
+        <input type="text" name="first_name" maxlength="30" required placeholder="  Your Name">
+        <input type="email" name="email" id="email" required placeholder="  Your Email Address">
+        <textarea name="message" id="text" maxlength="500" required placeholder="Write your message here"></textarea>
+        <button type="submit">Get in touch</button>
```
Comment (souad988) [inline index.html:220]: [ OPTIONAL ] Kindly do your best to replicate figma design as this button has a different background than in figma file 
**figma :**
get in touch white background

**Review by souad988 — APPROVED** [review]
Hi @Jawad-Nazari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔Good PR title and summary
✔Good communication with reviewer
✔Good collaboration with peer (commit history)
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Jawad-Nazari) [reply index.html:220]: **Hi @ souad988 ,**

Thanks for your time and for reviewing my code.  I have altered the button background color according to the Figma design.I do not know how I did not notice it. 

However many thanks for the approval and put this change optional. 🙏

**Pull request merged**